packages/@react-spectrum/form/stories/Form.stories.tsx

```diff
@@ -604,12 +605,14 @@ function FormWithSubmit() {
 
   return (
     <Form onSubmit={handleSubmit} isReadOnly={formStatus === 'valid'}>
-      {(formStatus === 'invalid' || formStatus === 'valid') &&
-        <InlineAlert variant={formStatus === 'invalid' ? 'negative' : 'positive'}>
-          <Header>{formStatus === 'invalid' ? 'Error' : 'Success'}</Header>
-          <Content>{formStatus === 'invalid' ? 'There was an error with the form.' : 'Form was successfully completed.'}</Content>
-        </InlineAlert>
-      }
+      <>
```

Comment:
as for why this is wrapped in a empty react fragment, see [stackoverflow comment](https://stackoverflow.com/questions/62612062/ts2322-false-element-is-not-assignable-to-type-reactelement-error-does-not)